### PR TITLE
Simplifies PaneControl code by removing unnecessary dynamic xaml.

### PR DESCRIPTION
**Pull request by cinqmilleans**
Simplifies PaneControl code by removing unnecessary dynamic xaml.

### CODE REVIEW

File: src/Files.Uwp/UserControls/Pane/PaneControl.xaml [hunk 1]
```diff
@@ -6,6 +6,6 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     d:DesignWidth="400" d:DesignHeight="300" mc:Ignorable="d">
 
-    <Grid x:Name="Panel" />
+    <ContentControl x:Name="Panel" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch" />
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/Pane/PaneControl.xaml:9]:
```suggestion
    <ContentControl x:Name="Panel"
    HorizontalContentAlignment="Stretch"
    VerticalContentAlignment="Stretch" />
```

**Comment by yaichenbaum** [comment]
@cinqmilleans is this ready for review?

**Comment by cinqmilleans** [comment]
@yair100 Yes it is ready for review. I just merged from main and retested to check.

File: src/Files.Uwp/UserControls/Pane/PaneControl.xaml (hunk 1 above)
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/Pane/PaneControl.xaml:9]: Can you format the xaml?

File: src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs [hunk 2]
```diff
@@ -8,34 +8,32 @@ namespace Files.Uwp.UserControls
 {
     public sealed partial class PaneControl : UserControl, IPane
     {
-        public PanePositions Position { get; private set; } = PanePositions.None;
-
-        private IPaneSettingsService PaneService { get; } = Ioc.Default.GetService<IPaneSettingsService>();
```
Comment (d2dyno1) [inline src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs:13]: We want to align with this format of defining services 🙂

Comment (cinqmilleans) [reply src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs:13]: I don't understand your comment. Could you explain?

Comment (d2dyno1) [reply src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs:13]: Can you keep the old `PaneService`? We want to avoid `_paneService` naming.

Comment (cinqmilleans) [reply src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs:13]: So I have to use a private field (not a property) with the name PaneService?

Comment (d2dyno1) [reply src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs:13]: No, just use
```csharp
private IPaneSettingsService PaneService { get; } = Ioc.Default.GetService<IPaneSettingsService>();
```

File: src/Files.Uwp/UserControls/Pane/PaneControl.xaml (hunk 1 above)
Comment (cinqmilleans) [reply src/Files.Uwp/UserControls/Pane/PaneControl.xaml]: It's done.

File: src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs (hunk 2 above)
Comment (cinqmilleans) [reply src/Files.Uwp/UserControls/Pane/PaneControl.xaml.cs:13]: It's done.

**Review by d2dyno1 — APPROVED** [review]

**Review by yair100 — APPROVED** [review]

**Pull request merged**